src/api/management/commands/backup_db.py

```
@@ -53,12 +57,22 @@ def handle(self, *args, **options):
+            success_msg=f"Successfully uploaded to s3://{BUCKET_NAME}/{s3_key}"
+            self.stdout.write(self.style.SUCCESS(success_msg))
 
             os.remove(local_path)
             self.stdout.write("Local cleanup done.")
 
+            BackupLog.objects.create(
+                status='SUCCESS',
+                message=success_msg
+            )
+
         except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)